packages/vaadin-lumo-styles/src/components/menu-bar-button.css

```diff
@@ -95,19 +95,19 @@
 
   /* prettier-ignore */
```

Comment:
Is this still relevant?

Reply:
I think the CSS selector wraps into second line as too long, which is why it's there.